### ISSUE TITLE
Wrong label tensor shape with pretrained_word_embeddings example

### ISSUE THREAD

**Issue by JimmyXiaodong** [fchollet/keras#6040]
There are 5 different labels in my dataset, but when I run this example on Keras 2.0.2 (using Tensorflow backend), it gives me a (None, 6) label tensor Shape. Why?
